frontends/p4/unusedDeclarations.cpp

```diff
@@ -172,6 +172,9 @@ bool RemoveUnusedDeclarations::isSystemFile(cstring file) {
     if (file.startsWith(p4includePath)) return true;
     // if the backend is invoked directly with '-I p4include'
     if (file.startsWith("p4include")) return true;
+    // In cases such as  "-I ./p4include", p4include may be within the path
+    if (file.find("p4include")) return true;
```

Comment:
perhaps it's better to check for `file.endsWith("p4include")`

Reply:
We can add another check with endsWith. Since in our case endsWith will not be executed because p4include is not at the end, because the `file `variable contains something like `"path"/p4include/core.p4`

Reply:
In order to get only the path from `file`, we will have to do cumbersome string processing to exclude the file name. 
Isn't it easier to leave only one check, for the presence of p4Include and remove `if (file.startsWith("p4include")) return true;` ?

Reply:
That's what I said.

Reply:
Okay then I'll make a change now